spec/controllers/course/assessment/submission/answer/programming/annotations_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
     let(:course) { create(:course, creator: user) }
     let(:assessment) { create(:assessment, :with_programming_question, course: course) }
     let(:submission) do
-      create(:course_assessment_submission, :submitted, assessment: assessment, user: user)
+      create(:submission, :submitted, assessment: assessment, user: user)
```

Comment:
Some of the factory names are actually quite long, like `course_assessment_answer_programming_file_annotation`, it takes almost one line if we use full form, that's the downside.